bin/codebasin

```diff
@@ -189,9 +202,14 @@ def main():
             )
         filtered_platforms.append(p)
 
-    # If no additional platforms are specified, a config file is required.
+    # A legacy config file is required if:
+    # - No additional platforms are specified; or
```

Comment:
I think I get the meaning being the comments, it's just that the implementation using `and` and it throws me off a little bit. Not sure if you want to refactor one or the other (maybe the wording of the docstring)

Reply:
Good catch.  I think I wrote the comment first, and worked through a few different implementations before landing on this one. Fixed in 1f5ad843764eae24173e4f72d2c9bafff44f6647.